dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/eperson/LoginChooser.java

```diff
@@ -232,26 +232,9 @@ public void addBody(Body body) throws SQLException, SAXException,
                                             : ("?" + request.getQueryString()));
                     loginURL = location.toString();
                 }
-					if (ShibAuthentication.class.getName().equals(authMethod.getClass().getName())) {
-						Item item = list.addItem();
-						item.addXref("#", message("Unified Login"), "signon label label-important");
-						//item.addFigure("./themes/UFAL/images/discojuice-logo.png", "#", "signon"); // comment to the link
-						item = list.addItem();
-						//Message discoComment = message("xmlui.EPerson.LoginChooser.discojuiceComment");
-						//item.addContent(discoComment);
-	
-						// Eduid link item = list.addItem();
-						//item.addFigure("./themes/UFAL/images/eduid-logo.png", loginURL, null);
-						//item.addXref(loginURL, "eduID.cz Login", "label label-important");
-						//final Message eduidComment = message("xmlui.EPerson.LoginChooser.eduidComment");
-						//item.addContent(eduidComment);
-					} else { // Normal Render
-						//final Item item = list.addItem();
-						//final Message otherwiseComment = message("xmlui.EPerson.LoginChooser.otherwiseComment");
-						// XXX not flexible if more auth methods
-						//item.addContent(otherwiseComment);
-						//item.addXref(loginURL, message(authTitle));
-					}
+                final Item item = list.addItem();
+                item.addXref(loginURL, message(authTitle));
```

Comment:
loginchooser is the handler for /xmlui/login with this change it add links to your configured authn methods; in this case to /xmlui/shibboleth-login and to /xmlui/password-login
example:
![image](https://user-images.githubusercontent.com/1842385/121157891-3a871b00-c84a-11eb-9689-aed10f87299a.png)
note you'd likely want to modify the messages.xml and have somethin more reasonable. Or you can go wild in the login chooser method add images and whatnot  (see the removed/comment code above)